client/src/api/jobs.ts

```diff
@@ -8,6 +8,10 @@ export type JobInputSummary = components["schemas"]["JobInputSummary"];
 export type JobDisplayParametersSummary = components["schemas"]["JobDisplayParametersSummary"];
 export type JobMetric = components["schemas"]["JobMetric"];
 
+export const NON_TERMINAL_STATES = ["new", "queued", "running", "waiting"];
+export const ERROR_STATES = ["error", "deleted"];
+export const TERMINAL_STATES = ["ok", "skipped"].concat(ERROR_STATES);
```

Comment:
The backend does define more job states, https://github.com/jmchilton/galaxy/blob/sample_sheets/lib/galaxy/model/__init__.py#L1663-L1675, worth adding the missing ones ? (plus paused)